stable/kubernetes-replicator/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- desaintmartin
+- tbd
```

Comment:
Do approvers/reviewers need to be project collaborators/org members? Otherwise, you're welcome to add me to the list.